output

```diff
@@ -0,0 +1,137 @@
+#include "swerve/Vision.h"
+#include "swerve/Drivetrain.h"
+#include <frc/DataLogManager.h>
+
+Vision::Vision(std::function<units::degree_t()> get_angle_fn)
+    : get_angle{get_angle_fn} {};
+
+std::vector<std::optional<frc::Pose2d>> Vision::get_bot_position()
+{
+  frc::SmartDashboard::PutNumber("vision step", 1);
+  std::vector<std::optional<frc::Pose2d>> ret;
+  // for (auto &i : m_photoncam_vec)
```

Comment:
Why commented out? Will we be using Photon Vision?

Reply:
This file is not used.